src/main/java/com/epam/reportportal/cucumber/AbstractReporter.java

```diff
@@ -54,6 +54,15 @@
 public abstract class AbstractReporter implements Formatter, Reporter {
 	private static final Logger LOGGER = LoggerFactory.getLogger(AbstractReporter.class);
 
+//  could be used in custom ScenarioReporter implementation like:
+//	public static void setCurrentLaunchId(String launchId)
+//	{
+//		customLaunchIdInUse = true;
+//		currentLaunchId = launchId;
+//	}
+	public static boolean customLaunchIdInUse;
```

Comment:
public static variables are not allowed since can be changed after launch start.

Reply:
@Vitovtx up